waspc/data/Generator/templates/server/src/routes/index.js

```diff
@@ -17,7 +17,16 @@ const router = express.Router()
 const middleware = globalMiddlewareConfigForExpress()
 
 router.get('/', middleware, function (_req, res) {
+  {=# isDevelopment =}
+  res.json({
+    message: 'Wasp Server is up and running!',
+    note: 'This is the Wasp server API. If you are looking for the web app, it is running on a separate port (usually 3000).',
+    documentation: 'https://wasp-lang.dev/docs'
+  });
```

Comment:
Some thoughts:
- It's JSON in dev, and an empty (I presume text) response in production - some people might make some assumption and try to parse it in production for their healthcheck? Maybe we stick with text here? 
- We intentionally removed the body on this endpoint because people complained it's less professional - they might again assume the prod response will look the dev response - we might point out that this is a dev only message?



```suggestion
  res.json({
    message: 'This is the Wasp server (development)',
    note: 'This is the Wasp server API. If you are looking for the web app, check the logs for Vite... You are seeing this message because the serve is running in develpoment mode, this will disappear when in production.',
    documentation: 'https://wasp-lang.dev/docs'
  });
```

I butchered it but I think you get the point I was trying to make